frontend/packages/dev-console/src/components/pipelineruns/__tests__/PipelineRunDetailsPage.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { PipelineRunModel } from '../../../models';
+import PipelineRunDetailsPage from '../PipelineRunDetailsPage';
+
+const pipelineRunDetailsProps: React.ComponentProps<typeof PipelineRunDetailsPage> = {
+  match: {
+    isExact: true,
+    path: `/k8s/ns/:ns/${referenceForModel(PipelineRunModel)}/:name/`,
+    url: 'k8s/ns/tekton-pipelines/tekton.dev~v1alpha1~PipelineRun/simple-pipeline-6wjqqw',
+    params: {
+      ns: 'tekton-pipelines',
+    },
+  },
+  kind: 'PipelineRun',
+};
+
+const expectedTabs: string[] = [`Overview`, `YAML`, `Logs`];
+
+describe('PipelineRun details page tests', () => {
+  it('should render `DetailsPage` Component with proper page tabs -> [ `Overview` , `YAML` , `Logs` ]', () => {
```

Comment:
I agree with Debsmitta's comment on the other file... we should probably not have the tabs listed out, it's a point of failure when the test needs to be updated.

```suggestion
  it('should render `DetailsPage` Component with proper page tabs', () => {
```